workers/generic-worker/os_groups_insecure_test.go

```diff
@@ -22,20 +21,15 @@ func TestEmptyOSGroups(t *testing.T) {
 	_ = submitAndAssert(t, td, payload, "completed", "completed")
 }
 
-func TestNonEmptyOSGroups(t *testing.T) {
+func TestNonEmptyOSGroupsUserIsNotIn(t *testing.T) {
 	setup(t)
 	payload := GenericWorkerPayload{
 		Command:    listGroups(),
 		MaxRunTime: 30,
-		OSGroups:   []string{"abc"},
+		OSGroups:   []string{"docker"},
 	}
 	defaults.SetDefaults(&payload)
 	td := testTask(t)
 
 	_ = submitAndAssert(t, td, payload, "exception", "malformed-payload")
-
-	logtext := LogText(t)
-	if !strings.Contains(logtext, "- osGroups: Array must have at most 0 items") {
```

Comment:
could we not keep a check that we at least get a snippet of a particular error message we were expecting (e.g. that we don't get completely the wrong message?)